nixos/modules/services/misc/ersatztv.nix

```diff
@@ -0,0 +1,126 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+
+let
+  inherit (lib)
+    mkIf
+    getExe
+    maintainers
+    mkEnableOption
+    mkOption
+    mkPackageOption
+    ;
+  inherit (lib.types) str path bool;
+  cfg = config.services.ersatztv;
+in
+{
+  options = {
+    services.ersatztv = {
+      enable = mkEnableOption "ErsatzTV";
+
+      package = mkPackageOption pkgs "ersatztv" { };
+
+      user = mkOption {
+        type = str;
+        default = "ersatztv";
+        description = "User account under which ErsatzTV runs.";
+      };
+
+      group = mkOption {
+        type = str;
+        default = "ersatztv";
+        description = "Group under which ErsatzTV runs.";
+      };
+
+      transcodeDir = mkOption {
+        type = path;
+        default = "/var/lib/etv-transcode";
+        description = ''
+          Temporary transcode directory
+        '';
+      };
+
+      configDir = mkOption {
+        type = path;
+        default = "/var/lib/ersatztv";
+        description = ''
+          Directory containing the server configuration files and SQLite Database file
+        '';
+      };
+
+      baseUrl = mkOption {
```

Comment:
does this need to be in the nix config or can it be in the etv config dir file(s)?

Reply:
It ends up as an environment variable on the service, so it needs to be in Nix